v2.0/backup.md

```diff
@@ -141,36 +151,42 @@ Per our guidance in the [Performance](#performance) section, we recommend starti
 ### Backup a Single Table or View
```

Comment:
Why do we reference views here?

Reply:
I don't know - this is an existing example that I haven't looked at.

Reply:
OK, please disregard, @Amruta-Ranade.